test/indexer/test_tdr.py

```diff
@@ -206,5 +226,67 @@ def _full_table_name(self, source: TDRSourceName, table_name: str) -> str:
         return source.bq_name + '.' + table_name
 
 
+class TestTDRClient(AzulTestCase):
+
+    def _mock_snapshot(self, access_token: str) -> JSON:
+        return {
+            'id': 'foo',
+            'name': f'{access_token}_snapshot'
+        }
+
+    def _mock_list_snapshots(self, method, url, **kwargs):
+        self.assertEqual(method, 'GET')
+        self.assertEqual(furl(url).remove(query=True).url,
+                         TDRClient()._repository_endpoint('snapshots'))
+        headers = kwargs.get('headers')
+        self.assertIsNotNone(headers)
+        try:
+            token = (headers.get('Authorization') or headers['authorization']).split('Bearer ').pop()
+        except (KeyError, ValueError):
+            assert False, headers
```

Comment:
I don't understand why you except and then assert False. Why not remove the try block all together and let the test fail if an exception is raised?

Reply:
This is detritus left over from debugging test failures that I neglected to clean up before pushing.